### ISSUE TITLE
Does not notify PAPO

### ISSUE THREAD

**Issue by solivier** [akeneo/michel-tag#37]
it does not notify PAPO since this PR https://github.com/akeneo/michel-tag/pull/36

**Comment by Doodoune** [comment]
Yes, I've an SDS ticket about that - https://akeneo.atlassian.net/browse/SDS-3950
